modelcontextprotocol/server.py

```diff
@@ -4,17 +4,17 @@
 from typing import Any, Dict, List
 from fastmcp import FastMCP
 from tools import (
-    search_assets,
-    get_assets_by_dsl,
-    traverse_lineage,
-    update_assets,
     create_glossary_category_assets,
     create_glossary_assets,
     create_glossary_term_assets,
     UpdatableAttribute,
     CertificateStatus,
     UpdatableAsset,
 )
+from tools.search import search_assets as search_assets_impl
```

Comment:
Let's add the _impl syntax for other tools as well, in order to keep the syntax consistent throughout

create_glossary_category_assets,
create_glossary_assets,
create_glossary_term_assets,